src/Core/Components/Camera/CameraGodRay.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "Utilities/ECS/Component.h"
+
+namespace MxEngine
+{
+	class GodRayEffect 
```

Comment:
it should be called 'CameraGodRay', in code and in editor to match other namings